core/cap-0046-06.md

```diff
@@ -275,7 +289,32 @@ fn create_token_from_contract(salt: Object) -> Result<Object, Error>;
 ```
 
 Uniqueness does not apply to non-wrapper
-token contracts, so this method uses the creating contract and a salt.
+token contracts, so this function uses the creating contract and a salt.
+
+### Semantics: AUTH_REQUIRED_FLAG
+
+On Stellar classic, any trustline created tied to an issuer that has
+`AUTH_REQUIRED_FLAG` set will be deauthorized on creation. Maintaining this
+requirement is important because it can be used to indicate a trustline that has
+gone through KYC verification. If we ignore `AUTH_REQUIRED_FLAG` in Soroban,
+then it would be possible for accounts, contracts, and ed25519 identifiers to
+interact with a token without going to KYC verification.
+
+There are two possible solutions proposed here-
```

Comment:
Shouldn't we settle on one?

Reply:
We should, but this is something I wanted to discuss for the protocol meeting. Would you prefer if I picked one and listed the other as an alternative?

Reply:
Then I'd just leave a note that the decision is TBD, because currently it's a bit confusing.

Reply:
I ended up just removing the auth_required section for now.